account_invoice_validation_queued/models/queue_job.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models
+
+
+class QueueJob(models.Model):
+    _inherit = 'queue.job'
+
+    def cancel(self):
```

Comment:
This code should never be there... Moreover it's a duplicate of  https://github.com/OCA/account-invoicing/commit/6ac79c298cb07f61f6f1888ef26eca702e4aa284#diff-01026d97e6f71aadeb837a145ddac1caR11

Reply:
Yes, because you can install each module and need the same feature, but there's no problem in having it "duplicated", and this is serving an specific UI purpose (to be able to cancel the job), so no better way to do it that I know of.